x-pack/plugin/downsample/qa/mixed-cluster/build.gradle

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import org.elasticsearch.gradle.Version
```

Comment:
I think the tests from `./x-pack/plugin/downsample/qa/rest` can be moved to `./x-pack/plugin/srx/yamlRestTest` and that way we would picky bag on the existing mixed cluster qa module? Instead of copying these tests into a new qa module?

I don't think we need to address this here, the important thing is to get this fix. Maybe in a followup?

Reply:
Yes, I will clean this up in a follow-up.